testdata/e2e/runTestValidate.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+
+curl --version
+
+echo "Test /api/validate should respond 200 if sealed secret is valid"
+
+SEALED_SECRET=$(curl --silent --show-error --request POST 'http://localhost/ssw/api/kubeseal' \
+  --header 'Accept: application/x-yaml' \
+  --data-binary '@stringData.yaml')
+
+echo "$SEALED_SECRET" | yq -r .apiVersion | grep --quiet "bitnami.com/v1alpha1"
+echo "$SEALED_SECRET" | yq -r .kind | grep --quiet "SealedSecret"
+echo "$SEALED_SECRET" | yq -r .metadata.name | grep --quiet "mysecretname"
+echo "$SEALED_SECRET" | yq -r .metadata.namespace | grep --quiet "mysecretnamespace"
+
+RESPONSE=$(curl --silent --show-error --request POST 'http://localhost/ssw/api/validate' \
+  --header 'Accept: text/plain' \
+  --data-binary "$SEALED_SECRET" \
+  --output /dev/null -w "%{http_code}" )
+
+echo "$RESPONSE" | grep --quiet 200
+
+echo "Test /api/validate should respond 400 if sealed secret is invalid"
+
+INVALID_SECRET=$(curl --silent --show-error --request POST 'http://localhost/ssw/api/kubeseal' \
+  --header 'Accept: application/x-yaml' \
+  --data-binary '@stringData.yaml' | yq '.metadata.name = "wrongname"')
+
+echo "$INVALID_SECRET" | yq -r .apiVersion | grep --quiet "bitnami.com/v1alpha1"
+echo "$INVALID_SECRET" | yq -r .kind | grep --quiet "SealedSecret"
+echo "$INVALID_SECRET" | yq -r .metadata.name | grep --quiet "wrongname"
+echo "$INVALID_SECRET" | yq -r .metadata.namespace | grep --quiet "mysecretnamespace"
+
+RESPONSE=$(curl --silent --show-error --request POST 'http://localhost/ssw/api/validate' \
+  --header 'Accept: text/plain' \
+  --data-binary "$INVALID_SECRET" \
+  --output /dev/null -w "%{http_code}" )
+
+echo "$RESPONSE" | grep --quiet 400
```

Comment:
The e2e test fails with the following error. (https://github.com/bakito/sealed-secrets-web/actions/runs/4959527967/jobs/8878502179)

The "continue-on-error" in the action does show the error, but it seems not to fail the action in the end.

Run ./runTests.sh
curl 7.81.0 (x86_6[4](https://github.com/bakito/sealed-secrets-web/actions/runs/4959527967/jobs/8878502179#step:8:5)-pc-linux-gnu) libcurl/7.81.0 OpenSSL/3.0.2 zlib/1.2.11 brotli/1.0.9 zstd/1.4.8 libidn2/2.3.2 libpsl/0.21.0 (+libidn2/2.3.2) libssh/0.9.6/openssl/zlib nghttp2/1.43.0 librtmp/2.3 OpenLDAP/2.[5](https://github.com/bakito/sealed-secrets-web/actions/runs/4959527967/jobs/8878502179#step:8:6).14
Release-Date: 2022-01-05
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv[6](https://github.com/bakito/sealed-secrets-web/actions/runs/4959527967/jobs/8878502179#step:8:7) Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL TLS-SRP UnixSockets zstd
Test /api/validate should respond 200 if sealed secret is valid
Error: Process completed with exit code 1.

Reply:
Running the test via the `runTests.sh`

```shell
curl 7.87.0 (x86_64-apple-darwin22.0) libcurl/7.87.0 (SecureTransport) LibreSSL/3.3.6 zlib/1.2.11 nghttp2/1.51.0
Release-Date: 2022-12-21
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtsp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS GSS-API HSTS HTTP2 HTTPS-proxy IPv6 Kerberos Largefile libz MultiSSL NTLM NTLM_WB SPNEGO SSL threadsafe UnixSockets
Test /api/validate should respond 200 if sealed secret is valid
curl: (7) Failed to connect to localhost port 80 after 2 ms: Couldn't connect to server
```

or `runTestValidate.sh`
```shell
curl 7.87.0 (x86_64-apple-darwin22.0) libcurl/7.87.0 (SecureTransport) LibreSSL/3.3.6 zlib/1.2.11 nghttp2/1.51.0
Release-Date: 2022-12-21
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtsp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS GSS-API HSTS HTTP2 HTTPS-proxy IPv6 Kerberos Largefile libz MultiSSL NTLM NTLM_WB SPNEGO SSL threadsafe UnixSockets
Test /api/validate should respond 200 if sealed secret is valid
curl: (7) Failed to connect to localhost port 80 after 3 ms: Couldn't connect to server
```

seems to fail locally as well

Reply:
```shell
2023/05/12 17:44:09 Connection to sealed secrets with (Cert URL: http://sealed-secrets.sealed-secrets.svc:8080/v1/cert.pem)
2023/05/12 17:44:09 Running sealed secrets web (e2e-tests) on port 8080
2023/05/12 17:44:09 | 200 |      34.801µs |      10.244.0.1 | GET      "/_health"
2023/05/12 17:44:15 | 200 |        23.4µs |      172.18.0.1 | GET      "/_health"
2023/05/12 17:44:15 | 200 |    4.299986ms |      172.18.0.1 | POST     "/api/kubeseal"
2023/05/12 17:44:15 Error in /api/validate: cannot get sealed secret service: resource name may not be empty
2023/05/12 17:44:15 | 400 |    1.202924ms |      172.18.0.1 | POST     "/api/validate"
```

Reply:
This seems still to be the issue @tiithansen was facing in his PR

with #146 the e2e test will now fail if the run tests had an error

Reply:
I’m trying to check locally but the last step of installing local sealed-secrets-web fails in a CrashLoopBackOff  with exit code 139 .. any idea?

Reply:
Do you have any logs of the pod?

Reply:
zero logs

Reply:
If I'm running the docker image outside k8s: `qemu: uncaught target signal 11 (Segmentation fault) - core dumped`
I'm on Apple M2